pom.xml

```diff
@@ -137,10 +137,15 @@
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
-			<groupId>com.liferay.portal</groupId>
-			<artifactId>portal-service</artifactId>
-			<version>6.1.0</version>
+		    <groupId>com.liferay.portal</groupId>
+		    <artifactId>com.liferay.portal.kernel</artifactId>
+		    <version>2.21.0</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+		  	<groupId>org.quartz-scheduler</groupId>
+		  	<artifactId>quartz</artifactId>
+			<version>2.2.3</version>
+		</dependency>
```

Comment:
I suppose that adding the quartz dependency was done in order to disable quartz monitoring.
But it would be simpler and better to change the following method in the JobInformations class:
```java
	private static boolean isQuartzAvailable() {
		try {
			Class.forName("org.quartz.Job");
			Class.forName("org.quartz.impl.SchedulerRepository");
			return true;
		} catch (final ClassNotFoundException e) {
			return false;
		}
	}
```